pkg/account/component.go

```diff
@@ -512,6 +512,7 @@ func (c *component) GetConfiguration(ctx context.Context, realmIDOverride string
 		AutoIdentificationAccountingEnabled:   adminConfig.AutoIdentificationAccountingEnabled,
 		AutoIdentificationPrepaymentRequired:  adminConfig.AutoIdentificationPrepaymentRequired,
 		AllowedBackURLs:                       allowedBackURLs,
+		OnboardingUserEditingEnabled:          config.OnboardingUserEditingEnabled,
```

Comment:
User can be logged into selfservice without being onboarded?

Reply:
It is more, your go through the SSE during onboarding